driver/monitoring/node/block_progress_test.go

```diff
@@ -136,13 +136,13 @@ func TestLocalRpcServer_CanHandleRequests(t *testing.T) {
 		t.Fatalf("failed to connect to server: %v", err)
 	}
 
-	var result string
+	var result map[string]any
```

Comment:
`map[string]any` is the expected type when parsing response from rpc call.
It could have also been a proper ethclient-related type (i.e. [Header](https://pkg.go.dev/github.com/ethereum/go-ethereum@v1.14.7/core/types#Header))  but epoch is specific to fantom so we need to parse the response generically.